go/libkb/login_state.go

```diff
@@ -187,11 +187,9 @@ func (s *LoginState) Shutdown() error {
 
 		if s.loginReqs != nil {
 			close(s.loginReqs)
-			s.loginReqs = nil // block future sends to this channel
```

Comment:
I hit a random test failure on this commit, so maybe that was the purpose.  I guess it's OK to send to a nil channel, but sending to a closed channel is a panic.

Hence, the subsequent PR which is to keep this behavior but also to get the close to work in the server loop.